### ISSUE TITLE
Through assosiations

### ISSUE THREAD

**Issue by colepanike** [balderdashy/sails-docs#321]
@mikermcneil  points out [here](http://youtu.be/yygXf1Idtbw?t=42m6s) that there is a major hole in the documentation. Through is supported but there's no documentation on it. I don't know enough about it's syntax to fill this hole or I would. Can someone give us at least a basic outline on this?

**Comment by rachaelshaw** [comment]
@particlebanana I know this is from awhile ago now; do you know if this stuff made it into the docs?

**Comment by sailsbot** [comment]
Thanks for posting, @colepanike.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
